site2/docs/deploy-bare-metal.md

```diff
@@ -48,20 +48,20 @@ For machines running a bookie and a Pulsar broker, we recommend using more power
 To get started deploying a Pulsar cluster on bare metal, you'll need to download a binary tarball release in one of the following ways:
 
 * By clicking on the link directly below, which will automatically trigger a download:
-  * <a href="pulsar:binary_release_url" download>Pulsar pulsar:version binary release</a>
+  * <a href="pulsar:binary_release_url" download>Pulsar {{pulsar:version}} binary release</a>
 * From the Pulsar [downloads page](pulsar:download_page_url)
 * From the Pulsar [releases page](https://github.com/apache/incubator-pulsar/releases/latest) on [GitHub](https://github.com)
 * Using [wget](https://www.gnu.org/software/wget):
 
 ```bash
-$ wget http://archive.apache.org/dist/incubator/pulsar/pulsar-pulsar:version/apache-pulsar-pulsar:version-bin.tar.gz
+$ wget pulsar:binary_release_url
```

Comment:
Should we use `{{...}}` as well for consistency with `{{pulsar:version}}`?

Reply:
We are going to change those in future patches for consistency. This one is just to make sure we got all the pulsar:version ones.

Reply:
Sure, no problem with that. In any case, I think that the notation `{{pulsar:version}}`, since it's common in many template languanges, would make it clearer to someone editing this document that this will be somehow replaced on the published version.